nodejs-assignment copy/node_modules/.bin/handlebars

```diff
@@ -0,0 +1 @@
+../handlebars/bin/handlebars
```

Comment:
We should not commit the node_modules folder. The package.json should be updated with the modules/packages that are used in the repo.